README.md

```diff
@@ -11,7 +11,7 @@ The recommended method of installation is [through composer](http://getcomposer.
 ```JSON
 {
     "require": {
-        "pschwisow/phergie-irc-plugin-react-url-shorten": "dev-master"
+        "pschwisow/phergie-irc-plugin-react-url-shorten": "^1"
```

Comment:
The caret syntax requires the newest version of composer. Not sure if we want to require that yet. Some people may be stuck with previous version of composer.